admin/metabox/class-metabox-form-tab.php

```diff
@@ -42,11 +52,13 @@ class WPSEO_Metabox_Form_Tab implements WPSEO_Metabox_Tab {
 	 * @param array  $options      Optional link attributes.
 	 */
 	public function __construct( $name, $content, $link_content, array $options = array() ) {
-		$this->name         = $name;
-		$this->content      = $content;
-		$this->link_content = $link_content;
-		$this->link_class	= isset( $options['link_class'] ) ? $options['link_class'] : '';
-		$this->link_title   = isset( $options['link_title'] ) ? $options['link_title'] : '';
+		$this->name            = $name;
+		$this->content         = $content;
+		$this->link_content    = $link_content;
+		$this->tab_class       = isset( $options['tab_class'] ) ? $options['tab_class'] : '';
```

Comment:
I think merging the options with a default options array would be more readable. For example:

```php
$default_options = array( 'tab_class' => '', 'link_class' => '', 'link_title' => '', 'link_aria_label' => '' );
$options = array_merge(  $default_options, $options );

$this->tab_class        = $options['tab_class'];
$this->link_class       = $options['link_class'];
$this->link_title          = $options['link_title'];
$this->link_aria_label = $options['link_aria_label'];
 
```